script/travis_build.sh

```diff
@@ -1,11 +1,7 @@
 #!/bin/bash -e
 
-cd "$1" || exit 1
-BUILD_NAME=$1
+BUILD_LOG="$(pwd)/travis.log"
```

Comment:
```suggestion
BUILD_LOG="$PWD/travis.log"
```

You can just use the PWD variable that the pwd command reads

Reply:
@1480c1 Just curious, is there any benefit with the diffrent way?

Reply:
String concatenation vs executing+collecting+string concatenation

The only time `$PWD` might not equal the output of `pwd` is if the `$PWD` variable contains `.` or `..`

<https://pubs.opengroup.org/onlinepubs/9699919799/utilities/pwd.html>